src/org/salesforce/apexdoc/ApexDoc.java

```diff
@@ -210,23 +213,32 @@ public static ClassModel parseFileContents(String filePath) {
                 // gather up our comments
                 if (strLine.startsWith("/*")) {
                     commentsStarted = true;
+                    if(strLine.startsWith("/**")){
+                    	lstComments.add(strLine);
```

Comment:
Good point, I didn't even see the repercussions of this change against single line doc blocks. That is an easy fix for me to make and test.